ferrocene/doc/internal-procedures/src/docs/signatures.rst

```diff
@@ -9,46 +9,63 @@ signed by the responsible parties, to attest they were reviewed and to prevent
-sign our qualification documents.
+.. note::
 
-To sign a document, you need to run this command inside of
-``ferrocene/ferrocene``:
+   While everyone can invoke the command to begin signing, after a signature is
+   made, the tooling will error out if you are not listed as an authorized
+   signer.
 
-.. code-block:: text
+.. caution::
 
-   ./x sign $path
+   If this is the first time you contribute to Ferrocene from this machine,
+   remember to :doc:`setup your local Ferrocene environment
+   <../setup-local-env>` before signing.
+
+We developed a small wrapper around the "cosign" tool to make it easy to
+sign our qualification documents. Cosign is fully managed by the Ferrocene build
+system: you don't need to install it.
 
-For example:
+To sign all of the Ferrocene documents in a single invocation, you can run:
 
 .. code-block:: text
 
-   ./x sign ferrocene/doc/evaluation-report
+   ./x sign
 
-Running the command will:
+Running the command will build the documentation locally, download the correct
+version of cosign, and start the signing process for each document. The signing
+process will:
 
-* Build the document locally to calculate its document ID and the hash of the
-  generated files.
+* As you whether you agree for your (company) email address to being published
+  in the Rektor transparency log. You need to agree to this.
 
-* Download the expected version of cosign, if it was not downloaded before.
+* Open the signature page in the browser (or, if the build system cannot open
+  your browser, show you the link to the page). In the signature page, you will
+  need to select **Microsoft** as your identity provider, and authenticate with
+  your company credentials.
 
-* Invoke cosign to digitally sign the document metadata gathered earlier.
+* Verify the signature locally, and persist it to disk.
 
-When cosign is invoked, you will need to:
+The signing process will be repeated *separately* for each document you need to
+sign. Once ``./x sign`` finishes the signature process will be completed, and you
+can commit and push the changes.
 
-* Agree to your company email address being published in the Rektor transparency
-  log.
+Signing a single document
+-------------------------
 
-* In the browser window that opened, select "Microsoft".
+It's possible to provide one or more paths to ``./x sign`` to sign just those
+documents. For example, if you just want to sign the internal procedures, you
+can run:
 
-* Authenticate with your company email credentials.
+.. code-block::
 
-This will generate an ephemeral code signing certificate for your company email
-address and sign the contents of the document with it. Once that's done, commit
-the new files generated by the signing tool.
+   ./x sign ferrocene/doc/internal-procedures
+
+When doing the initial signature for a release it's recommended to sign all the
+documents at the same time. Signing documents individually is meant for the
+following signatures in the same branch.
```

Comment:
what _following_ signatures

Reply:
Expanded the sentence in the new commit.